datafusion/core/src/datasource/avro_to_arrow/arrow_array_reader.rs

```diff
@@ -1536,12 +1536,10 @@ mod test {
         .unwrap()
         .resolve(&schema)
         .unwrap();
-        let r4 = apache_avro::to_value(serde_json::json!({
-            "col1": null
-        }))
-        .unwrap()
-        .resolve(&schema)
-        .unwrap();
+        let r4 = apache_avro::to_value(serde_json::json!({ "col1": null }))
```

Comment:
these changes because I run `./ci/scripts/rust_fmt.sh` and it has been changed....